avro/pom.xml

```diff
@@ -47,7 +47,7 @@ abstractions.
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <version>1.8.2</version>
+      <version>1.9.2</version>
```

Comment:
I was under impression that this will break about everything... did you run unit test suite?

But even if not, we need this to be in separate PR, with separate issue.

Reply:
I was wary of that upgrade as well, but doing that version bump and running the avro test suite resulted in passing tests. 

We're a gradle shop so running stuff with maven was a bit hokey with all our firewall rules.   Readying through #167 looks like some stuff has been upgraded but I imagine there must be missing unit tests for some of the underlying avro lib changes.

Reply:
Ok, that's odd wrt tests: for me code did not compile with missing imports.
And compilation on CI and locally did not work.

I wish I was more up to date with Avro lib changes since there are newer minor versions, some of which might resolve issues -- although from what I remember, things were quite messy wrt backwards-compatibility (lack thereof) for some specific usage.